### ISSUE TITLE
Templates. Elements should have sufficient color contrast

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#61]

**Comment by sastels** [comment]
This is grey1 in `tailwind.config.js` that we use all over the place in similar situations (that are probably also breaking a11y). We can darken that grey1 up to #595959 which will meet WCAG AAA for normal text on a white background.
